ts-sdk/whirlpool/tests/increaseLiquidity.test.ts

```diff
@@ -1,5 +1,480 @@
-import { describe } from "vitest";
+import { describe, it, beforeAll } from "vitest";
+import {
+  increaseLiquidityInstructions,
+  openPositionInstructions,
+  openFullRangePositionInstructions,
+} from "../src/increaseLiquidity";
+import { rpc, signer, sendTransaction } from "./utils/mockRpc";
+import { setupMint, setupAta } from "./utils/token";
+import {
+  fetchMaybePosition,
+  fetchPosition,
+  getPositionAddress,
+} from "@orca-so/whirlpools-client";
+import { fetchToken } from "@solana-program/token-2022";
+import { address, assertAccountExists, type Address } from "@solana/web3.js";
+import assert from "assert";
+import {
+  setupPosition,
+  setupTEPosition,
+  setupWhirlpool,
+} from "./utils/program";
+import {
+  DEFAULT_FUNDER,
+  setDefaultFunder,
+  SPLASH_POOL_TICK_SPACING,
+} from "../src/config";
+import {
+  setupAtaTE,
+  setupMintTE,
+  setupMintTEFee,
+} from "./utils/tokenExtensions";
 
-describe.skip("Increase Liquidity", () => {
-  // TODO: <-
+describe("Increase Liquidity Instructions", () => {
+  const tickSpacing = 64;
+  const tokenBalance = 1_000_000n;
+  let mintA: Address;
+  let mintB: Address;
+  let mintTEA: Address;
+  let mintTEB: Address;
+  let mintTEFee: Address;
+  let ataMap: Record<string, Address> = {};
+  let whirlpools: Record<string, Address> = {};
```

Comment:
Took me a little bit to figure out what is what here. Maybe it makes sense to do:
```
let ataMap: Record<string, Address> = {}; // key is the mint address
let poolAToB: Address
let poolAToTEA: Address
// etc.
let equallySpacedPositions:  Record<string, Address> = {}; // key is the whirlpool address
let oneSidedPositionsA:  Record<string, Address> = {}; // key is the whirlpool address
let oneSidedPositionsB:  Record<string, Address> = {}; // key is the whirlpool address
```

Reply:
The pools you are only ever getting with a static string (not dynamic) so there is no strict need for them to be in an object

Reply:
Makes sense